librz/main/rz-find.c

```diff
@@ -541,8 +554,13 @@ RZ_API int rz_main_rz_find(int argc, const char **argv) {
 			rz_list_append(ro.keywords, (void *)opt.arg);
 			break;
 		case 'E':
-			ro.mode = RZ_SEARCH_ESIL;
-			rz_list_append(ro.keywords, (void *)opt.arg);
+			ro.quiet = true;
+			if (opt.arg) {
+				ro.exec_command = opt.arg;
+				rz_list_append(ro.keywords, (void *)opt.arg);
+			} else {
+				ro.mode = RZ_SEARCH_ESIL;
+			}
```

Comment:
no. this command cannot be both.

Reply:
as i told you previously remove the esil search from here.

Reply:
My bad, I misunderstood it since I had removed the ESIL search earlier, I thought you want me to have both. Will Fix it.